### PR TITLE
calculateRentalCost solution

### PR DESCRIPTION
**Pull request by DidenkoDenis306**

### CODE REVIEW

**Review by generatedman — APPROVED** [review]
good job!

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -28,7 +28,16 @@
...
+  const cost = days * 40;
+
+  switch (true) {
+    case days >= 3 && days < 7:
+      return cost - 20;
+    case days >= 7:
+      return cost - 50;
+    default:
+      return cost;
+  }
```
Comment (naziamoff) [inline src/calculateRentalCost.js:40]: Switch (true) is kind of a crutch. Let's try to rewrite it to `if` statements

**Review by naziamoff — CHANGES_REQUESTED** [review]
not bad, but let's change the way of implementation

**Review by 33Orange — APPROVED** [review]
Well done!

For self improve:
You can make all hardcode values ​​into constants (price multiplier, discounts) so that they can be easily changed in the future.
```
const multiplier = 40;
const cost = days * multiplier;
```

**Review by naziamoff — APPROVED** [review]

**Pull request closed without merging**